docker/hive/Dockerfile

```diff
@@ -6,7 +6,7 @@ FROM ghcr.io/dojoengine/dojo:v1.0.0-alpha.0 as katana
 # Indexer service
 ### Apibara DNA indexer and indexer
 FROM quay.io/apibara/starknet:1.5.0 as apibara
-FROM quay.io/apibara/sink-mongo:0.7.0 as indexer
+FROM quay.io/apibara/sink-mongo:0.7.1 as indexer
```

Comment:
In this same file, I think we have this line also to modify no?

```
COPY --from=indexer /nix/store/${APIBARA_SINK_BIN_DIR}-apibara-sink-mongo-0.7.0/bin/apibara-sink-mongo /usr/local/bin/sink-mongo
```

to

```
COPY --from=indexer /nix/store/${APIBARA_SINK_BIN_DIR}-apibara-sink-mongo-0.7.1/bin/apibara-sink-mongo /usr/local/bin/sink-mongo
```

Reply:
Missed that, thanks!